Editor/SettingsAnalysis/QualitySettingsAnalyzer.cs

```diff
@@ -43,6 +45,22 @@ class QualitySettingsAnalyzer : ISettingsModuleAnalyzer
             "The <b>Async Upload Buffer Size</b> option for one or more quality levels in the project's Quality Settings is set to the default value.",
             "If the project encounters long loading times when loading large amount of texture and/or mesh data, experiment with increasing this value to see if it allows content to be uploaded to the GPU more quickly. This is most likely to help if you are loading large textures. Note that this setting controls a buffer size in megabytes, so exercise caution if memory is limited in your application.");
 
+        static readonly Descriptor k_TextureStreamingDisabledDescriptor = new Descriptor(
+            PAS0030,
+            "Quality: Texture streaming disabled",
+            Area.Quality,
+            "The <b> Texture Streaming </b> is disabled . More mipmap textures will be loaded into memory on the GPU.",
```

Comment:
Just minor letters I would remove:

"<b>Texture Streaming</b> is disabled."

(I'd say we don't need "The" at the start, and then there are a few extra spaces we can remove)